values/nginx-ingress-controller/values.daemonset.yaml

```diff
@@ -0,0 +1,80 @@
+# Normally, NodePort will listen to traffic on all nodes, and uses kube-proxy
```

Comment:
This file now looks great, with all the explanations. Could you make this the default? By "making it the default", I mean actually moving this to `charts/nginx-ingress-controller/values.yaml`. That way, we will not have any file under values/nginx-ingress-controller at all, and installing with `helm install wire/nginx-ingress-controller` will provide one working solution out-of-the-box.